packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -33,6 +33,8 @@ import '../common/wait.dart';
 import '_extension_io.dart' if (dart.library.html) '_extension_web.dart';
 import 'wait_conditions.dart';
 
+export 'wait_conditions.dart';
```

Comment:
Is this needed for the docs fixes?

Reply:
Yeah, the docs reference stuff from there, it wasn't part of the public API though and no docs were generated for it.

Reply:
Alternatively, we could change the docs to not reference this. Let me check that this should actually be part of the public API...

Reply:
You're right, just exporting this for the docs is strange since it isn't really part of the public API. I instead changed the location where it was referenced to not link it.

Reply:
SGTM